src/platform/mtk/include/platform/lib/dai.h

```diff
@@ -0,0 +1,10 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright(c) 2024 Google LLC.  All rights reserved.
+ * Author: Andy Ross <andyross@google.com>
+ */
+#ifndef _SOF_PLATFORM_MTK_LIB_DAI_H
+#define _SOF_PLATFORM_MTK_LIB_DAI_H
+
```

Comment:
With https://github.com/thesofproject/sof/pull/9653 , this can be dropped if only supporting Zephyr builds.